decouple.py

```diff
@@ -216,7 +216,7 @@ def _find_file(self, path):
 
         # search the parent
         parent = os.path.dirname(path)
-        if parent and parent != os.path.abspath(os.sep):
+        if parent and os.path.normcase(parent) != os.path.normcase(os.path.abspath(os.sep)):
```

Comment:
alternatively, this can also be done by doing:
```suggestion
        if parent and os.path.dirname(parent) != parent:
```